addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiSensorBaseHandler.java

```diff
@@ -66,10 +69,13 @@ void calculateBatteryLevelFromVoltage(Integer voltage) {
         Integer battLevel = (int) ((float) (voltage - VOLTAGE_MIN_MILLIVOLTS)
                 / (float) (VOLTAGE_MAX_MILLIVOLTS - VOLTAGE_MIN_MILLIVOLTS) * 100);
         updateState(CHANNEL_BATTERY_LEVEL, new DecimalType(battLevel));
-        if (battLevel <= BATT_LEVEL_LOW) {
+        int lowThreshold = isBatteryLow ? BATT_LEVEL_LOW + BATT_LEVEL_LOW_HYS : BATT_LEVEL_LOW;
```

Comment:
yes - users already reported, that the battery values are not going down steadily. perhaps depending on temperature or whatever, it decreases a few mV, then increases with the next report.
If this happens around the threshold, it would be activated and deactivated.
The intention of this change is to introduce a hysteresis on the threshold to prevent it from toggling.